### PR TITLE
Add playbook-fac-services-docker-images

### PR DESCRIPTION
**Pull request by xresende**
This PR supersedes PR https://github.com/lnls-sirius/lnls-ansible/pull/263

### CODE REVIEW

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Review by lerwys — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -103,6 +104,7 @@ $(playbook_TARGETS): %: %.yml
 
 -include Makefile_services.mk
 -include Makefile_reboot.mk
+-include Makefile_deploy.mk
```
Comment (lerwys) [inline Makefile:107]: It seems the Makefile name is different. It's missing the ".mk" suffix

**Review by lerwys — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (xresende) [reply Makefile:107]: fixed

**Review by lerwys — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

File: group_vars/all [hunk 2]
```diff
@@ -101,6 +101,12 @@ pkg_version_hlacon_conda_pyqt: "5.12.3"
 pkg_version_hlacon_conda_pydm: "1.10.4"
 pkg_version_hlacon_conda_siriushlacon: "2.0.13"
 
+# === fac docker services ===
+
+pkg_version_docker_machine_applications: "master"
```
Comment (fernandohds564) [inline group_vars/all:106]: @xresende, why are we using the master branch here?

**Review by fernandohds564 — APPROVED** [review]

File: group_vars/all (hunk 2 above)
Comment (xresende) [reply group_vars/all:106]: at its final version, these repos are stateless: they simply take states (pkg_versions) defined in ansible configs and apply to docker images, recreating new versions to be deployed. I think adding versioning to these repos is not natural and only adds maintenance and elaboration into the deploy process.

Comment (xresende) [reply group_vars/all:106]: at the end, the way I devised the deploy process, everything will be managed from ansible makefile targets: deploy into desktops, IOC and csconsts docker images, and ansible deploy tag creation.

**Review by lerwys — APPROVED** [review]

**Pull request merged**